frappe/__init__.py

```diff
@@ -292,6 +292,7 @@ def init(site: str, sites_path: str = ".", new_site: bool = False, force=False)
 	local.role_permissions = {}
 	local.valid_columns = {}
 	local.new_doc_templates = {}
+	local.system_notifications_queue = []
```

Comment:
can't this just be part of enqueue to flush after commit? 

It implements same exact behavior:
- Queue till commit
- flush after commit
- clear queue in case of rollback or no commit